src/Controller/Organization/UserListController.php

```diff
@@ -27,22 +29,20 @@ public function __construct(OrganizationRepository $organizationRepository, User
         $this->organizationRepository = $organizationRepository;
     }
 
-    public function __invoke(Request $request): Response
+    public function __invoke(Request $request, Organization $organization): Response
     {
-        $organization = $this->getUser();
-
-        if (!$organization instanceof Organization) {
+        $currentOrganization = $this->getUser();
+        if (!$currentOrganization instanceof Organization) {
             throw new AccessDeniedException();
         }
 
-        $childOrganizations = [];
-        if (null === $organization->parent) {
-            $childOrganizations = $this->organizationRepository->findByParent($organization);
+        if ($currentOrganization !== $organization && $organization->parent !== $currentOrganization) {
+            throw new AccessDeniedException('You cannot manage this organization');
         }
 
-        return $this->render('organization/user-list.html.twig', [
+        return $this->render('organization/user/user-list.html.twig', [
             'organization' => $organization,
-            'users' => $this->userRepository->findByOrganizations(array_merge([$organization], $childOrganizations)),
+            'users' => $this->userRepository->findByOrganization($organization),
```

Comment:
This will make DT75 for examepl, to not be able to edit users related to child structures, is this expected ?

Reply:
There is a new "List users for this organization" button in the orga list